LazarevDV/lab_4/lab_4.cpp

```diff
@@ -0,0 +1,120 @@
+#include <iostream>
+#include <conio.h>
+
+
+using namespace std;
+
+float power() {
+    float number, counter = 1; //number число вводимое в степень, pow сама степень, counter счетчик
+    int pow;
+    cout << "Введите число и степень через пробел: ";
```

Comment:
Function in that lab work should not ask user to input anything. You should ask them before function call and pass it as function arguments